ArjunaJTS/integration/src/main/java/com/arjuna/ats/jbossatx/BaseTransactionManagerDelegate.java

```diff
@@ -165,7 +238,7 @@ public Transaction suspend()
     public boolean containsValue(final TransactionLocal transactionLocal, final Transaction transaction) {
         com.arjuna.ats.jta.transaction.Transaction transactionImple = (com.arjuna.ats.jta.transaction.Transaction) transaction;
         if(transactionImple.isAlive()) {
-            return (transactionImple.getTxLocalResource(transactionLocal) != null ? true : false);
+            return (transactionImple.getTxLocalResource(transactionLocal) != null);
```

Comment:
what has this got to do with listeners :) 
(maybe split it to a separate commit if you really absolutely must)